eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -1,38 +1,71 @@
 param(
+  # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
+  [Parameter(Mandatory = $true)]
+  $RepoNameWithOwner,
+  # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
   [Parameter(Mandatory = $true)]
```

Comment:
We cannot make both sets of these parameters mandatory. I think we will need to make both optiona.

Reply:
Sure. Done.